bin/offline-helm.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
 set -euo pipefail
+set -x
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ANSIBLE_DIR="$( cd "$SCRIPT_DIR/../ansible" && pwd )"
+ansible-playbook -i "$ANSIBLE_DIR"/inventory/offline "$ANSIBLE_DIR"/helm_external.yml --skip-tags=rabbitmq-external -vv
```

Comment:
Why did we need to move this here? `bin/offline-helm.sh` and `bin/offline-cluster.sh` are both called by `bin/offline-deploy.sh` in order?. :thinking:

Reply:
Previously when I did not had the individual tag for the rabbitmq-external role in the helm_external.yml. I had to do it this way, because offline-cluster.sh script is also used by us to create a test env in the Hetzner machine and we wanted to avoid the rabbitmq-external role there.

But now after the recent changes, I have safely removed it from offline-helm and moved back to offline-cluster.sh